deploy/crds/app.kiegroup.org_v1alpha1_kogitoapp_cr.yaml

```diff
@@ -8,4 +8,6 @@ spec:
       uri: https://github.com/kiegroup/kogito-examples
       contextDir: process-quarkus-example
     # set your maven nexus repository
-    #mavenMirrorURL: http://nexus3-nexus.apps-crc.testing/repository/maven-public/
\ No newline at end of file
+    #mavenMirrorURL: http://nexus3-nexus.apps-crc.testing/repository/maven-public/
+    # enable to print maven transfer logs in console
+    #maven-download-output: true
```

Comment:
could we have it like `mavenMirrorURL`, I mean it should be `mavenDownloadOutput` field in all examples

Reply:
+1

Reply:
changed name to enableMavenDownloadOutput